virt/virt-4-13-release-notes.adoc

```diff
@@ -301,4 +301,8 @@ $ oc patch storageprofile <storage_profile> --type=merge -p '{"spec": {"claimPro
 * In a heterogeneous cluster with different compute nodes, virtual machines that have HyperV Reenlightenment enabled cannot be scheduled on nodes that do not support timestamp-counter scaling (TSC) or have the appropriate TSC frequency. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2151169[*BZ#2151169*])
 
 //CNV-28577
-* If you deploy {VirtProductName} with {rh-storage-first}, you must create a dedicated storage class for Windows virtual machine disks. See link:https://access.redhat.com/articles/6978371[Optimizing ODF PersistentVolumes for Windows VMs] for details.
\ No newline at end of file
+* If you deploy {VirtProductName} with {rh-storage-first}, you must create a dedicated storage class for Windows virtual machine disks. See link:https://access.redhat.com/articles/6978371[Optimizing ODF PersistentVolumes for Windows VMs] for details.
```

Comment:
Actually, we discovered, that it's needed not only for Windows, but for all Virtual Machines.

Reply:
Thanks for bringing this to our attention! It looks like it would be better to address this in a different PR/Jira so I'll go ahead and open a bug to get this fixed.